### DISCUSSION TITLE
part-ii/2015-40/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#135] [Question comments]
Part II 2015 Differential Geometry: Paper 1, Section II, $22 G$ 

https://questions.tripos.org/part-ii/2015-40/

**Comment by ArthurConmy** [comment]
This is a bit of a strange question, it's short anyway and all that is asked for is an "indication of a proof"?!

I believe the final answer is that yes, the result continues to hold. <a href="https://en.wikipedia.org/wiki/Isoperimetric_inequality#On_a_plane">Wikipedia</a> seems to agree.

I argued that we can put disjoint sufficiently small circles around each point of non-differentiability, and then connect the two incoming lines with the minor arc of a circle (provided we localise enough) like so:

![](https://i.imgur.com/9ibhU9U.png)

Then I hope not much justification need be done, but we get a region $ \tilde{\Omega} $ for which we can approximate $ | \ell ( \tilde{\Omega} ) - \ell ( \Omega ) |< \varepsilon$ and also $ | A( \tilde{\Omega} ) - A( \Omega )  | < \varepsilon$ for any $\varepsilon > 0$ and of course the isoperimetric inequality does hold for $ \tilde{\Omega} $, and so it will hold for $ \Omega $ too.
